### PR TITLE
Feature/retweet routes

### PR DESCRIPTION
**Pull request by megh-khaire**

### CODE REVIEW

**Review by msomya — APPROVED** [review]

File: backend/controllers/follows.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 const User = require("./../models/User");
 const Sequelize = require("sequelize");
 
+const formatUsers = (users) => {
+  return users.map((user) => {
+    user.isFollowedByUser = user.isFollowedByUser === 1 ? true : false;
```
Comment (Saunved) [inline backend/controllers/follows.js:6]:
```suggestion
    user.isFollowedByUser = user.isFollowedByUser === 1;
```

File: backend/controllers/tweet.js [hunk 2]
```diff
@@ -15,6 +16,13 @@ const formatTweet = (tweetObj) => {
     formattedTweet.isLikedByUser = formattedTweet.isLikedByUser === 0 ? false : true;
```
Comment (Saunved) [inline backend/controllers/tweet.js:16]:
```suggestion
    formattedTweet.isLikedByUser = !formattedTweet.isLikedByUser === 0;
```

File: backend/controllers/tweet.js [hunk 3]
```diff
@@ -15,6 +16,13 @@ const formatTweet = (tweetObj) => {
...
   }
 
+  if ("isRetweetedByUser" in formattedTweet) {
+    formattedTweet.isRetweetedByUser = formattedTweet.isRetweetedByUser === 0 ? false : true;
```
Comment (Saunved) [inline backend/controllers/tweet.js:20]:
```suggestion
    formattedTweet.isRetweetedByUser = !formattedTweet.isRetweetedByUser === 0;
```

**Review by Saunved — APPROVED** [review]
Feel free to fix these minor changes later

**Pull request merged**